src/etl/do_etl.py

```diff
@@ -37,141 +43,159 @@ class DOETL(ETL):
 
             syns = []
 
             local_id = None
-            defLinksUnprocessed = []
-            defLinksProcessed = []
+            def_links_unprocessed = []
+            def_links_processed = []
             subset = []
             definition = ""
             is_obsolete = "false"
-            ident = k
+            ident = key
             prefix = ident.split(":")[0]
 
-
             if "meta" in node:
                 if "synonyms" in node["meta"]:
                     syns = [s["val"] for s in node["meta"]["synonyms"]]
                     for synonym in syns:
-                        doSynonym = {
-                            "primary_id": k,
+                        do_synonym = {
+                            "primary_id": key,
                             "synonym": synonym
                         }
-                        do_synonyms_list.append(doSynonym)
+                        do_synonyms_list.append(do_synonym)
 
                 if "basicPropertyValues" in node["meta"]:
-                    altids = [s["val"] for s in node["meta"]["basicPropertyValues"]]
-                    for alt_id in altids:
+                    alt_ids = [s["val"] for s in node["meta"]["basicPropertyValues"]]
+                    for alt_id in alt_ids:
                         if "DOID:" in alt_id:
-                            secondaryId = {
-                                "primary_id": k,
+                            secondary_id = {
+                                "primary_id": key,
                                 "secondary_id": alt_id
                             }
-                            do_altids_list.append(secondaryId)
+                            do_alt_ids_list.append(secondary_id)
 
                 if "xrefs" in node["meta"]:
-
                     o_xrefs = node["meta"].get('xrefs')
                     if o_xrefs is not None:
-                        for xrefIdDict in o_xrefs:
-                            xrefId = xrefIdDict["val"]
-                            if ":" in xrefId:
-                                local_id = xrefId.split(":")[1].strip()
-                                prefix = xrefId.split(":")[0].strip()
-                                complete_url = ETLHelper.get_complete_url_ont(local_id, xrefId)
-                                generated_xref = ETLHelper.get_xref_dict(local_id, prefix, "ontology_provided_cross_reference", "ontology_provided_cross_reference", xrefId, complete_url, xrefId + "ontology_provided_cross_reference")
+                        for xref_id_dict in o_xrefs:
+                            xref_id = xref_id_dict["val"]
+                            if ":" in xref_id:
+                                local_id = xref_id.split(":")[1].strip()
+                                prefix = xref_id.split(":")[0].strip()
+                                complete_url = ETLHelper.get_complete_url_ont(local_id, xref_id)
+                                generated_xref = ETLHelper.get_xref_dict(\
```

Comment:
can the first item in the list be in line with the function call, I find this more readable than the \

Reply:
Yes this I can see how you would prefer this - I will make it happen.